### DISCUSSION TITLE
Documentation for gifos

### DISCUSSION THREAD

**Discussion by arunoruto** [x0rzavi/github-readme-terminal#5] [Q&A]
Awesome project! But I was having some difficulties, and documentation would have been helpful!
If you need help setting [mkdocs](https://www.mkdocs.org/) up, I can try to help you :)

**Comment by x0rzavi** [comment]
> Awesome project! But I was having some difficulties, and documentation would have been helpful! If you need help setting [mkdocs](https://www.mkdocs.org/) up, I can try to help you :)

Hi there. Sure! any kind of help is most appreciated.

**Reply by FelipeKras** [reply]
Would be nice to see a **contributing.md** for better guidance.

**Reply by x0rzavi** [reply]
guidelines were added, please take a look :)
